youtube_dl/extractor/polskieradio.py

```diff
@@ -15,6 +16,60 @@
 )
 
 
+class PolskieRadioProgrammeIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?polskieradio\.pl/(?P<id>\d+/\d+)(,.*)?$'
```

Comment:
This does not match URLs like `http://www.polskieradio.pl/7/5102?foo=bar` that work fine in browser.